CUDADataFormats/Track/interface/PixelTrackHeterogeneous_t.h

```diff
@@ -64,13 +63,12 @@ namespace pixelTrack {
   constexpr uint32_t maxNumber() { return 32 * 1024; }
 #endif
 
-  using TrackSoA = TrackSoAT<maxNumber()>;
-  using TrajectoryState = TrajectoryStateSoA<maxNumber()>;
+  using TrackSoA = TrackSoAHeterogeneousT<maxNumber()>;
+  using TrajectoryState = TrajectoryStateSoAT<maxNumber()>;
   using HitContainer = TrackSoA::HitContainer;
-  using Quality = trackQuality::Quality;
 
 }  // namespace pixelTrack
 
 using PixelTrackHeterogeneous = HeterogeneousSoA<pixelTrack::TrackSoA>;
```

Comment:
I would move this to a separate file with the old name `CUDADataFormats/Track/interface/PixelTrackHeterogeneous.h`:
```c++
#ifndef CUDADataFormats_Track_PixelTrackHeterogeneous_h
#define CUDADataFormats_Track_PixelTrackHeterogeneous_h

#include "CUDADataFormats/Common/interface/HeterogeneousSoA.h"
#include "CUDADataFormats/Track/interface/TrackSoAHeterogeneousT.h"

using PixelTrackHeterogeneous = HeterogeneousSoA<pixelTrack::TrackSoA>;

#endif  // #ifndef CUDADataFormats_Track_PixelTrackHeterogeneous_h
```
and leave the downstream `#include`s unchanged.

Reply:
Done in latest commit.